backend/api/v1/routes/metrics.py

```
@@ -0,0 +1,172 @@
+            "notification_sent": None,
+            "error": None,
+            "retry_count": 0,
+            "final_answer": None
+        })
+        
+        # SQL 캐싱
+        if result.get('generated_sql') and not result.get('error'):
+            saved.sql_query = result['generated_sql']
+            saved.sql_generated_at = datetime.utcnow()
+            repository.update_metric(saved.id, saved)
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)